templates/types/streaming/express/src/controllers/chat.controller.ts

```diff
@@ -24,8 +25,12 @@ export const chat = async (req: Request, res: Response) => {
 
     // retrieve document ids from the annotations of all messages (if any)
     const ids = retrieveDocumentIds(messages);
-    // create chat engine with index using the document ids
-    const chatEngine = await createChatEngine(ids, data);
+    // create chat engine with index using the document ids and additional params passed in the request body
```

Comment:
The disadvantage here is that it complicates the example code

Reply:
Yes, after checking the type, I kept this file as is, which is easier for the user to understand.